controllers/news.controller.js

```diff
@@ -0,0 +1,24 @@
+const {
+  readTopics, readArticles } = require("../models/news.model");
+  
+  const getObjects = (req, res, next) => {
+    readTopics().then((data) => {
+      res.status(200).send(data);
+    })
+    .catch((err) => {
+      console.log(err);
+      next(err);
+    });
+  };
+  
+
+  const getarticles = (req, res, next) => {
+    readArticles().then((data) => {
+      res.status(200).send(data);
```

Comment:
Please think about the naming of the key you return the data on - something more descriptive like _articles_ would be more appropriate 👍